<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bankomatwindow</class>
 <widget class="QWidget" name="bankomatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="balanceButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Показать баланс</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="takeBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>591</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Снять</string>
   </property>
   <widget class="QPushButton" name="takeButton_100">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ₽</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeButton_500">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>500 ₽</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeButton_1000">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>1000 ₽</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeButton_other">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Другая сумма</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fillButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>309</y>
     <width>300</width>
     <height>430</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пополнить</string>
   </property>
  </widget>
  <widget class="QLabel" name="godbyeText">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>700</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>До свидания!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
